README.md

```diff
@@ -94,8 +94,8 @@ We provide a high performance [packet generator](simulator) to emulate the RRU.
    * In another terminal, run  `./build/sender --num_threads=2 --core_offset=1 --frame_duration=5000 --enable_slow_start=1 --conf_file=data/tddconfig-sim-ul.json` to start the emulated RRU 
      with uplink configuration.
    * The above steps use Linux networking stack for packet I/O. Agora also supports using DPDK
-   to bypass the kernel for packet I/O. To enable DPDK, run `cmake -DUSE_DPDK=1 ..; make -j` to 
-   rebuild code for Mellanox NICs; for Intel NICs, run `cmake -DUSE_DPDK=1 -DUSE_MLX_NIC=0 ..; make -j` 
+   to bypass the kernel for packet I/O. To enable DPDK, run `cmake .. -DUSE_DPDK=1; make -j` to 
+   rebuild code for Mellanox NICs; for Intel NICs, run `cmake .. -DUSE_DPDK=1 -DUSE_MLX_NIC=0; make -j` 
```

Comment:
Not necessary but works either way.  I changed this to force the CI to run again, so moving back to the original is not a bad idea either.

Reply:
Changing it back to the original might be better to be consistent with the rest of README.